libsolidity/codegen/ABIFunctions.h

```diff
@@ -116,27 +128,6 @@ class ABIFunctions
 		std::string toFunctionNameSuffix() const;
 	};
 
-	/// @returns the name of the cleanup function for the given type and
-	/// adds its implementation to the requested functions.
-	/// @param _revertOnFailure if true, causes revert on invalid data,
-	/// otherwise an assertion failure.
-	std::string cleanupFunction(Type const& _type, bool _revertOnFailure = false);
```

Comment:
Hm, can we ignore this part before #5815 is merged? I can work on it later the week, Friday earliest.

Reply:
Sure, I can just keep it in ABIFunctions for now.